dsms/knowledge/kitem.py

```diff
@@ -380,6 +388,34 @@ def validate_summary(cls, value: Union[str, Summary]) -> Summary:
             value = Summary(kitem=cls, text=value)
         return value
 
+    @field_validator("hdf5")
```

Comment:
Should all these specific functions related to hdf5 be inside the kitem.py file rather than keep them separate?

Reply:
For now, we give the opprtunity to the user to let it him update the overall hdf5-property and hence the validation would be needed in the `kitem.py`. Making the update column-wise would be more complicated and might not give a clear benefit for now, even if it would distribute the specific hdf5-functions to the `HDF5Container`-model instead of the `kitem.py`.